src/scss/components/checkout/paymentProvider/_paymentProvider.scss

```diff
@@ -6,6 +6,7 @@
     height: fontSize("large");
 
     + .paymentProviderHeader-name {
+        font-size: 18px;
```

Comment:
How come we need to adjust font-size here?

Reply:
Before we used picture with payment method name, but now we need to handle and display different branding names so this font size looks the best. If you have any suggestion where it should be - let me know.